api/v1alpha3/virtualmachine_types.go

```diff
@@ -197,6 +197,19 @@ const (
 	// ManagerID on a VirtualMachine contains the UUID of the
 	// VMware vCenter (VC) that is managing this virtual machine.
 	ManagerID = GroupName + "/manager-id"
+
+	// +kubebuilder:validation:Enum=ImportVM;RegisterVM
+	//
+	// SourceAPI on a VirtualMachine contains the API responsible for creating / updating
+	// 	the VirtualMachine custom resource for a virtual machine that already exists on the
+	// infrastructure. The ImportVM API is used to import a traditional virtual machine in
```

Comment:
`s/The ImportVM API is used to import a traditional virtual machine in/The ImportVM API is used to import a traditional virtual machine into/`